backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -111,11 +108,14 @@
         expect(page).to have_content("Default")
       end
 
-      it "will show the variant sku" do
-        order = create(:completed_order_with_totals)
-        visit spree.edit_admin_order_path(order)
-        sku = order.line_items.first.variant.sku
-        expect(page).to have_content("SKU: #{sku}")
+      context "with a completed order" do
+        let!(:order) { create(:completed_order_with_totals) }
```

Comment:
I think it's better having this inlined inside of the it block:
```rb
order = create(:completed_order_with_totals)
```

It's always better to have the code contained in the single place, than trying to figure out where did this variable come from (especially if `let!` is used and not `let`).